sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/eventhubs/core/EventHubsTemplate.java

```diff
@@ -30,7 +30,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EventHubsTemplate.class);
 
     private final EventHubProducerFactory producerFactory;
-    private EventHubMessageConverter messageConverter = new EventHubMessageConverter();
+    private final EventHubMessageConverter messageConverter = new EventHubMessageConverter();
```

Comment:
I prefer not to make this final, it's not used right now, but it provides the possibility to extend the default EventHubMessageConverter.

Reply:
And also keep the setMessageConveter

Reply:
updated